### PR TITLE
Update 0000-01-01-intro.md

### PR DESCRIPTION
**Pull request by akr7205**
changed line 5

### CODE REVIEW

File: _posts/0000-01-01-intro.md [hunk 1]
```diff
@@ -2,5 +2,5 @@
 layout: slide
 title: "Welcome to our slide deck!"
 ---
-
```
Comment (akr7205) [inline _posts/0000-01-01-intro.md:5]: changed line 5

**Review by akr7205 — COMMENTED** [review]
looks good

**Review by akr7205 — COMMENTED** [review]
change require

**Review by akr7205 — COMMENTED** [review]
changes the line 5

**Review by arunrout8249 — APPROVED** [review]
okay done

**Pull request merged**